lib/messaging/messagehandler.rb

```diff
@@ -22,7 +28,7 @@ class MessageHandler
 
   def initialize
     #@dbaccess = DbAccess.new
-    @dbaccess = $dbaccess
+    @dbaccess = $dbaccess || NullDatabase.new
```

Comment:
A stub to stop nil exceptions in the case that it can't connect to the DB. @TimEvWw might have some more input on a better solution, but it works for now.